src/frontend/src/flows/register/finish.ts

```diff
@@ -92,10 +94,56 @@ export const displayUserNumberTemplate = ({
   });
 };
 
+// A non-exported wrapper around HTMLImageElement, to ensure this exact image is loaded
+// (a no other image is passed as an argument to this page)
+class IdentityBackground {
+  public img: HTMLImageElement;
+
+  constructor() {
+    const img = new Image();
+    img.src = `${BASE_URL}image.png`; // Setting the src kicks off the fetching
+    img.classList.add("c-input--id__art");
+    img.alt = "";
+    this.img = img;
+  }
+}
+
 export const displayUserNumberPage = renderPage(displayUserNumberTemplate);
 
-export const displayUserNumber = (userNumber: bigint): Promise<void> => {
+// Omit specified functions parameters, for instance OmitParams<..., "foo" | "bar">
```

Comment:
Technically, it omits properties from the first argument rather than parameters, right?